content/en/integrations/guide/azure-native-manual-setup.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Azure Native Integration Manual Setup Guide
+kind: guide
+description: "Steps for manually setting up the Datadog Azure native integration"
+further_reading:
+- link: "https://docs.datadoghq.com/agent/faq/why-should-i-install-the-agent-on-my-cloud-instances/"
+  tag: "Documentation"
+  text: "Why should I install the Datadog Agent on my cloud instances?"
+- link: "https://docs.datadoghq.com/integrations/guide/azure-portal/"
+  tag: "Documentation"
+  text: "Datadog in the Azure Portal"
+- link: "https://www.datadoghq.com/blog/azure-government-monitoring-datadog/"
+  tag: "Blog"
+  text: "Monitor Azure Government with Datadog"
+- link: "https://www.datadoghq.com/blog/azure-container-apps/"
+  tag: "Blog"
+  text: "Monitor Azure Container Apps with Datadog"
+- link: "https://www.datadoghq.com/blog/how-to-monitor-microsoft-azure-vms/"
+  tag: "Blog"
+  text: "How to Monitor Microsoft Azure VMs"
+- link: "https://www.datadoghq.com/blog/azure-app-service-datadog-serverless-view/"
+  tag: "Blog"
+  text: "Explore Azure App Service with the Datadog Serverless view"
+---
+
+{{< site-region region="us3" >}}
+
+## Overview
+
+Use this guide to manually set up the Datadog Azure Native integration through creation of the Datadog resource in Azure.
+
+The Datadog resource streamlines management and data collection for your Azure environment. Datadog recommends using this method when possible. This replaces the App Registration credential process used by the standard Azure integration for metric collection and Event Hub setup for log forwarding.
+
+### Prerequisites
+
+#### Required permissions
+
+To set up the Azure Native integration, you must be an **Owner** on any Azure subscriptions you want to link, and **Admin** for the Datadog organization you are linking them to. Ensure you have the appropriate access before starting the setup.
+
+## Setup
+
+Configuring the Azure integration requires the creation of a Datadog resource in Azure. These resources represent the connection or link between a Datadog organization and an Azure subscription. A Datadog resource is needed for each subscription you wish to monitor with Datadog. 
```

Comment:
This statement isn't correct now that we have multiple subscriptions - One resource can now be used to connect multiple subscriptions to Datadog